components/textbook-beta/DynamicBackgroundLogo.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div ref="canvasWrapper" class="dynamic-background-logo">
+    <canvas ref="canvas" class="dynamic-background-logo-background-canvas" />
+    <SketchedLogo class="dynamic-background-logo-overlay" />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+
+// These numbers are the count of the results of executing a quantum circuit:
+//
+// qreg q[5];
+// creg c[5];
+//
+// h q[0];
+// cx q[0],q[1];
+// cx q[0],q[2];
+// x q[0];
+// cx q[0],q[3];
+// cx q[0],q[4];
+//
+// There are only two possible results: 11001 and 00110.
+// The quantum noise and decoherence create some errors.
+// That is the reason ACCUMULATED_SUCCESS_RATIO is different than 1.
+// The SUCCESS_RATIO_1 is used as the probability to draw a big square
+// The SUCCESS_RATIO_2 is used as the probability to draw a small square
+// An unsuccessful result draws a blank space.
```

Comment:
A multiple-line comment should use the `/* */` style [per spec](https://tc39.es/ecma262/#prod-MultiLineComment). `//` is for single line comments.